cedar-policy-core/src/entities.rs

```diff
@@ -174,7 +174,7 @@ impl Entities {
             // We do this before adding the actions, because we trust the
             // actions were already validated as part of constructing the
             // `Schema`
-            let checker = EntitySchemaConformanceChecker::new(schema, extensions);
+            let checker = EntitySchemaConformanceChecker::new(schema, extensions.clone());
```

Comment:
is it possible for `EntitySchemaConformanceChecker` to hold an `&Extensions` instead?  (I do recognize that `Extensions` is still pretty cheap to clone, so this is a minor nit)

Reply:
Yes. I'm confident I could get rid of all the clones and just do refs everywhere, but it creates a bunch of changes, so I didn't go through with it

Reply:
Sounds fine

Reply:
not so bad actually #1100